hoverctl/wrapper/hoverfly.go

```diff
@@ -188,8 +189,45 @@ func SetMiddleware(target Target, binary, script, remote string) (v2.MiddlewareV
 	return apiResponse, nil
 }
 
+func GetLogs(target Target, format string) ([]string, error) {
+	headers := map[string]string{
+		"Content-Type": "text/plain",
+	}
+
+	if format == "json" {
+		headers["Content-Type"] = "application/json"
+	}
+
+	response, err := doRequest(target, "GET", v2ApiLogs, "", headers)
+	if err != nil {
+		return []string{}, err
+	}
+
+	defer response.Body.Close()
+
+	responseBody, _ := ioutil.ReadAll(response.Body)
```

Comment:
So the `/api/v2/logs` endpoint is doing that, but its using the `Content-Type` header rather than the `Accept`. I will raise a separate PR and fix that now.

When you ask for logs in a JSON format from `/api/v2/logs`, should they come back in a valid JSON object or should they be coming across coming back as plaintext but as JSON objects?. For example:
```
{
    "logs": [
        {
            "level":"info",
            "method":"GET",
            "msg":"started handling request",
            "remote":"[::1]:61802",
            "request":"/api/v2/logs",
            "time":"2017-05-16T11:26:22+01:00"
        },
        {
            "level":"info",
            "method":"GET",
            "msg":"started handling request",
            "remote":"[::1]:61802",
            "request":"/api/v2/logs",
            "time":"2017-05-16T11:26:22+01:00"
        }
    ]
}
```
```
{"level":"info","method":"GET","msg":"started handling request","remote":"[::1]:61802","request":"/api/v2/logs","time":"2017-05-16T11:26:22+01:00"}
{"level":"info","method":"GET","msg":"started handling request","remote":"[::1]:61802","request":"/api/v2/logs","time":"2017-05-16T11:26:22+01:00"}
```